components/NotionTypes/Callout/Callout.jsx

```diff
@@ -11,8 +11,14 @@ export const Callout = ({ columnItem }) => {
 
     return (
         <div className={styles.Callout__container}>
-            <Image className={styles.Callout__icon} width={25} height={25} src={iconUrl} />
-            <p>{text}</p>
+            <Image
+                className={styles.Callout__icon}
+                width={25}
+                height={25}
+                src={iconUrl}
+                alt="icon"
```

Comment:
Let's set empty `alt=""`. 

Not an empty alt would be useful to us only if we could describe the content of the picture.